segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -408,5 +441,10 @@ private void logError(String segment, Throwable u) {
         }
     }
 
+    @Override
+    public void close() {
+        // Todo: Delete Transient Segments.
+    }
```

Comment:
(This is a question, not a request for a change)
This ties the lifetime to the physical connection, rather than that of the writer / session. So on the client side if an